src/chapters/inheritance/studio.rst

```diff
@@ -0,0 +1,79 @@
+.. _inheritance-studio:
+
+Studio: Inheritance
+===================
+
+For this studio, you will design and build a console program that allows
+the user to take a quiz. This means you will have to create some
+questions, and get some input from the user.
+
+First, the questions. We want to be able to handle multiple types of
+questions:
+
+#. **Multiple choice**: a question with a fixed set of possible answers, of which only one may be chosen and only one answer is correct
+#. **Checkbox**: a question with a fixed set of possible answers, of which any number may be chosen; there is one correct combination of choices
+#. **True/False**: a question that has a true/false answer
+
+Design
+------
+
+In order to design your program, consider:
+
+#. What do these types of questions have in common?
+#. What makes these question types different?
+
+First, design a base class (called ``Question``) that contains the common features, and
+design subclasses for each of the question types. For each question type
+be sure to include: 
+
+#. Class name
+#. Fields and properties with access modifiers
+#. Methods with access modifiers
+#. Any inheritance relationship
+
+Should any of the question classes be abstract? If so, should any of its
+methods be abstract?
+
+Make sure that there is functionality included to display the questions,
+to display the possible answers, and to check to see if the answer(s) is
+correct.
+
+Then design the ``Quiz`` class. A quiz has a list of questions, and we
+should be able to: 
+
+#. Add questions
+#. Run or carry out the quiz
+#. Grade the quiz
+
+Implementation
+--------------
+
+Code the classes that you designed above in a new IntelliJ project. If you like, get some feedback
```

Comment:
just a thought, we could think about telling them to add this to a 'studios' folder in java-web-dev-exercises. again, because it can be tricky to start a project from scratch. but then again, they will probably be better off getting familiar with the workflow at some point. but if we want to go in that direction, i'd give them some more guidance on how to create a project from scratch since they wont have done it since one of the first lessons

Reply:
oh! and maybe we can talk abotu adding some testing requirements here as well

Reply:
They have had to start a project from scratch in multiple lessons since the first one, so they should be used to it by now.